### PR TITLE
EZP-30725: Added db upgrade script for CT drafts cleanup

### PR DESCRIPTION
**Pull request by pawbuj**
| Question           | Answer
| ------------------ | ------------------
| **JIRA issue**     | [EZP-30725](https://jira.ez.no/browse/EZP-30725)
| **Bug/Improvement**| yes
| **New feature**    | no
| **Target version** | `7.5`
| **BC breaks**      | no
| **Tests pass**     | yes
| **Doc needed**     | no

Adds some SQLs that remove from existing DB, drafts of non-existing users

**TODO**:
- [ ] Implement feature / fix a bug.
- [ ] Implement tests.
- [ ] Fix new code according to Coding Standards (`$ composer fix-cs`).
- [x] Ask for Code Review.

### CODE REVIEW

File: data/update/mysql/dbupdate-7.5.2-to-7.5.3.sql [hunk 1]
```diff
@@ -1,3 +1,46 @@
 UPDATE ezsite_data SET value='7.5.3' WHERE name='ezpublish-version';
 
 ALTER TABLE ezcontentclass_attribute MODIFY data_text1 VARCHAR(255);
+
+--
+-- EZP-30725: Removes content type drafts of no existing users
```
Comment (webhdx) [inline data/update/mysql/dbupdate-7.5.2-to-7.5.3.sql]: non existing users?

File: data/update/postgres/dbupdate-7.5.2-to-7.5.3.sql [hunk 2]
```diff
@@ -1,3 +1,46 @@
 UPDATE ezsite_data SET value='7.5.3' WHERE name='ezpublish-version';
 
 ALTER TABLE ezcontentclass_attribute ALTER COLUMN data_text1 TYPE varchar(255);
+
+--
+-- EZP-30725: Removes content type drafts of no existing users
```
Comment (webhdx) [inline data/update/postgres/dbupdate-7.5.2-to-7.5.3.sql]: Same here

File: data/update/mysql/dbupdate-7.5.2-to-7.5.3.sql (hunk 1 above)
Comment (pawbuj) [reply data/update/mysql/dbupdate-7.5.2-to-7.5.3.sql]: thx for catching this

**Comment by pawbuj** [comment]
> Preconditions: admin1 and admin2 users exist in application
> Steps:
> 
> 1. admin1 creates Content Type
> 2. admin2 goes to edit mode of the Content Type and log out (it's blocked now)
> 3. admin1 removes admin2
> 4. run the script
> 5. Logged in as admin1 go to edit mode of Content Type
> 
> Actual result: draft is not removed. Error message is displayed that Content Type is blocked by "another user"
> Expected result: draft should be removed.

it is fixed, please also note that after running script you should clear the cache because of caching in the persistence layer.

**Comment by mnocon** [comment]
@pawbuj Shoudn't the same change be made in Postgresql scripts?

**Comment by lserwatka** [comment]
You can merge it up.

**Comment by pawbuj** [comment]
merged

**Pull request merged**